packages/react-examples/src/react-experiments/UnifiedPeoplePicker/UnifiedPeoplePicker.WithEdit.Example.tsx

```diff
@@ -376,6 +393,12 @@ export const UnifiedPeoplePickerWithEditExample = (): JSX.Element => {
         defaultDragDropEnabled={false}
         onKeyDown={_onKeyDown}
         onValidateInput={_onValidateInput}
+        itemListAriaLabel="Recipient list"
```

Comment:
On itemListAriaLabel I am getting this TS error: 
```
Type '{ componentRef: MutableRefObject<any>; selectedItemsListProps: ISelectedPeopleListProps<IPersonaProps>; floatingSuggestionProps: IFloatingPeopleSuggestionsProps; ... 7 more ...; headerComponent: Element; }' is not assignable to type 'IntrinsicAttributes & Pick<IUnifiedPickerProps<IPersonaProps>, "className" | "onPaste" | "onFocus" | "onBlur" | ... 15 more ... | "onValidateInput">'.
  Property 'itemListAriaLabel' does not exist on type 'IntrinsicAttributes & Pick<IUnifiedPickerProps<IPersonaProps>, "className" | "onPaste" | "onFocus" | "onBlur" | ... 15 more ... | "onValidateInput">'.ts(2322)

```
it works correctly in Storybook I can see the Aria label correctly set, but I am not sure why in the Editor I have this in Problem.